website/content/docs/agent/index.mdx

```diff
@@ -141,7 +141,7 @@ These are the currently-available general configuration option:
 
 - `cache` <code>([cache][caching]: <optional\>)</code> - Specifies options used for Caching functionality.
 
-- `listener` <code>([listener][listener]: <optional\>)</code> - Specifies the addresses and ports on which the Agent will respond to requests.
+- `listener` <code>([listener][listener]: <optional\>)</code> - Specifies the addresses and ports on which the Agent will respond to requests. Ignored if a `cache` configuration block is not declared. 
```

Comment:
Because this behavior is unintuitive, it might be worth adding a brief description of why a listener requires a cache even when cache functionality will not be used.